### DISCUSSION TITLE
View logs in google cloud

### DISCUSSION THREAD

**Discussion by Elepert** [NightscoutFoundation/xDrip#3253] [Q&A]
I've deployed nightscout to google cloud following these instructions: https://navid200.github.io/xDrip/docs/Nightscout/GoogleCloud.html I then deployed from my github fork and that works as well. 

When I run my code locally with `npm start` I see a bunch of output and my `console.log` statements are printed there. How can I see those same logs on my google cloud instance? I've checked `var/log/nginx` but that doesn't have them.

**Comment by Navid200** [comment]
We put together Google Cloud initially and there were complaints that it was too complicated for someone who has no experience with Unix.
So, we added dialog to make it use menus instead of the user having to use the shell.
For someone who needs Nightscout, it works and everything they need is what they see through the menus.

You have to decide what you want.
If you want to run Nightscout your way, you can set it up yourself.
If you want to use our setup, tell us what the problem is you are experiencing that is not explained how to deal with in the guide and we either add an option to the menu or show you what you have missed in the guide.

I hope that makes sense.

**Reply by Elepert** [reply]
I see, I'm developing my own plugin and wanted to test it in google cloud, so there isn't a problem that I'm dealing with, just typical development bugs.

**Reply by Navid200** [reply]
Linux dialog is something I learned in order to set all the menus up.  I discovered that if I did not clear the screen, one dialog would overlap the next.  So, I clear the screen right before creating a dialog.  Clearing the screen wipes all the logs that are shown in the shell.

If I'm not mistaken, the logs are kept in a repository.  So, if you search, you should find how to access logs from it.  Sorry I am not sure how myself.
